packages/editor/test/reducers/model.schema.test.ts

```diff
@@ -0,0 +1,32 @@
+import { createEditorStore } from '../../src/helpers/util';
+import { addModelSchema, getModelSchema } from '../../src/reducers';
+import { JsonSchema } from '@jsonforms/core';
+
+describe('Adding Model Schema Tests', () => {
+  test('Add model schema during store initialization', () => {
+    const modelSchema: JsonSchema = {
+      type: 'object',
+      properties: {
+        name: {
+          type: 'string'
+        }
+      }
+    };
+    const store = createEditorStore({}, {}, {}, {}, {}, {}, {}, {}, {}, modelSchema);
```

Comment:
Do we have an issue to track refactoring the `createEditorStore` function? If so, ignore, otherwise please create one (as we discussed recently we need definitely re-work this).

Reply:
I opened the issue https://github.com/eclipsesource/jsonforms-ui-schema-editor/issues/18